cmd/libsnap-confine-private/string-utils-test.c

```diff
@@ -74,6 +74,27 @@ static void test_sc_startswith(void)
 	g_assert_false(sc_startswith("ba", "bar"));
 }
 
+static void test_sc_is_beneath(void)
+{
+	// Essential difference from sc_startswith
+	g_assert_true(sc_is_beneath("/foo/bar", "/foo"));
+	g_assert_false(sc_is_beneath("/foo/bar", "/foo2"));
+	g_assert_true(sc_is_beneath("foo/bar", "foo"));
+	// True cases
+	g_assert_true(sc_is_beneath("/foo", "/foo"));
```

Comment:
Hmm? API-wise, this seems like it should be false.

Reply:
Perhaps the name "beneath" is incorrect, in my original intent it was about a subset of paths that are rooted at /foo. This obviously includes /foo in this sense (it's not a proper subset). The name did not convey this though.